### ISSUE TITLE
Tree view drop down should show only the parents and children of passed value.

### ISSUE THREAD

**Issue by ShilpaVKulkarni** [leovo2708/ngx-treeview#211]
Hi,

I have drop down tree view as follows. 
<img width="561" alt="drop down 1" src="https://user-images.githubusercontent.com/29088209/56488404-e864cd80-64fb-11e9-89b1-1bdc83294cab.png">
I want to show only the parents and children of passed value. 
For example if I pass value as **Hubli** then it should show as follows:
<img width="561" alt="drop down 2" src="https://user-images.githubusercontent.com/29088209/56488428-f6b2e980-64fb-11e9-82d6-1ed024771f34.png">

If I type filter text box then it is working properly but I want to do this with code. I tried using onFilterChange() and also with filteredChange event emitter but not getting result as expected.

How to achieve this?

**Comment by ShilpaVKulkarni** [comment]
@leovo2708 can you please provide solution for this?

**Comment by leovo2708** [comment]
What happens with filtering? I see it is working as expected.

**Comment by ShilpaVKulkarni** [comment]
Yes filtering is working properly when we type text in filter bar. 
But when I try to call   **onFilterChanged()** inside my function without typing text in filter bar at that time the  **onFilterChanged()** is not working as expected.

**Comment by ShilpaVKulkarni** [comment]
My drop down tree view is like this:
https://user-images.githubusercontent.com/29088209/56488404-e864cd80-64fb-11e9-89b1-1bdc83294cab.png

But I want to show only the tree view as follows:
https://user-images.githubusercontent.com/29088209/56488428-f6b2e980-64fb-11e9-82d6-1ed024771f34.png

I want to show only the parent and child locations of **Hubli**.
If I type **Hubli** in filter bar I am getting tree view as I expected. 
But I need to do this with code instead of using filter bar.
So I tried calling **onFilterChanged()**method  inside my function but it is not working as I expected (filtering is not working).
Please give solution to achieve this.
